docs/configuration.md

```diff
@@ -142,15 +121,19 @@ N.B. In some instances a configuration option may be marked **optional** but con
 version: 0.1
 ```
 
-The version option is **required** and indicates the version of the configuration being used. It is expected to remain a top-level field, to allow for a consistent version check before parsing the remainder of the configuration file. 
+The `version` option is **required**. It specifies the configuration's version.
+It is expected to remain a top-level field, to allow for a consistent version
+check before parsing the remainder of the configuration file. 
 
-N.B. The version of the registry software may be found at [/version/version.go](https://github.com/docker/distribution/blob/master/version/version.go)
+>**Note**: You can find the registry software version at
+[/version/version.go](https://github.com/docker/distribution/blob/master/version
```

Comment:
This file doesn't actually provide the version until after the linker runs. It only contains a reasonable guess.